debian/cozy-standalone.postrm

```diff
@@ -44,6 +44,12 @@ case "$1" in
 		[ -d /usr/local/var/log/cozy ] && echo "Delete /usr/local/var/log/cozy directory" && rm -rf /usr/local/var/log/cozy
 		[ -d /usr/local/cozy ] && echo "Delete /usr/local/cozy directory" && rm -rf /usr/local/cozy
 		[ -d /var/www/cozy-letsencrypt ] && echo "Delete /var/www/cozy-letsencrypt directory" && rm -rf /var/www/cozy-letsencrypt
+		
+		echo "Delete Cozy's UNIX users"
+		users=`cat /etc/passwd | cut -d":" -f1 | grep "^cozy\-"`
+		for user in $users; do
+			userdel $user
```

Comment:
The homes directory are located in `/usr/local/cozy/apps`, which is already removed [here](https://github.com/babolivier/cozy-debian/blob/054d05fc464bf48d0cfa8efd62a5d0c32779e6fd/debian/cozy-standalone.postrm#L45)